proto/tendermint/statesync/message_test.go

```diff
@@ -204,8 +205,8 @@ func TestStateSyncVectors(t *testing.T) {
 						AppVersion: 11,
 					},
 					Synchrony: &tmproto.SynchronyParams{
-						MessageDelay: 550,
-						Precision:    90,
+						MessageDelay: durationPtr(550),
+						Precision:    durationPtr(90),
```

Comment:
Is there any reason these fields should be pointers? My understanding's that `time.Duration` is just an `int64`.

Reply:
This is a side-effect of the `nullable` option being set as it is.

Reply:
Does it need to be set as it is? :slightly_smiling_face: Just trying to understand the reasoning.